frontend/src/features/customer/components/cards/HelperSelectInfoCard.tsx

```diff
@@ -0,0 +1,17 @@
+import { IcAlertCircle } from '@icons';
+
+const HelperSelectInfoCard = () => {
+  return (
+    <div className='bg-neutral-10 flex flex-col gap-[1rem] rounded-[0.8rem] p-[1.2rem]'>
```

Comment:
rounded = 0.6rem인 것 같아요!

```suggestion
    <div className='bg-neutral-10 flex flex-col gap-[1rem] rounded-[0.6rem] p-[1.2rem]'>
```